src/balena-model.ts

```diff
@@ -364,6 +364,10 @@ export interface Release {
 	start_timestamp: DateString;
 	end_timestamp: DateString | null;
 	update_timestamp: DateString;
+	is_passing_tests: boolean;
+	release_type: 'final' | 'draft';
+	is_child_of__release: { __id: number } | [Release];
```

Comment:
Let's not add this and `pull_request_url ` just yet, the whole thing about child releases and pull requests is a bit unclear as of now. We are still not decided on how all of it will work as part of the transformers work, so I'd rather wait for us to finalize how everything will work there before we add it to our balena.io model

Reply:
sounds good, that's step 2 anyway. updated.